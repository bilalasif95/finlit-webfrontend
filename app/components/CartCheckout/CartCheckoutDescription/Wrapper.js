import styled from 'styled-components';

const Wrapper = styled.div`
  .cart_checkout {
    .header {
      align-tems: center;
      display: flex;
      justify-content: flex-end;
      .input-group {
        width: 30%;
        .form-control {
          border: none;
          border-right: 1px solid #e4e4e4;
          font-size: 1rem;
          padding: 0.25rem 0.75rem;
          &:focus {
            box-shadow: none;
          }
        }
        button {
          background-color: transparent;
          border-color: transparent;
          color: #8b8b8b;
          padding: 0.25rem 0.5rem;
          position: absolute;
          right: 0;
          top: -2px;
          z-index: 4;
          &:focus {
            box-shadow: none;
          }
        }
        .btn-secondary {
          background-color: transparent;
          border-color: transparent;
        }
      }
      .cart {
        align-items: center;
        display: flex;
        justify-content: center;
        position: relative;
        width: 7%;
        svg {
          color: #8b8b8b;
          height: 1rem;
          width: 1rem;
        }
        .no_item {
          align-tems: center;
          display: flex;
          justify-content: center;
          background-color: #404041;
          border-radius: 2rem;
          color: #fff;
          font-size: 0.75rem;
          height: 1.25rem;
          padding: 1.1px 0 0;
          position: absolute;
          top: 0.1rem;
          right: 0.25rem;
          width: 1.25rem;
        }
      }
    }
    table {
      width: 100%;
      tr {
        align-tems: center;
        display: flex;
        justify-content: space-between;
        border-bottom: 1px solid #e4e4e4;
      }
      th,
      td {
        font-size: 0.875rem;
        font-weight: 400;
        padding: 0.7rem 0;
        width: 20%;
      }
      th {
        color: #404041;
      }
      td {
        color: #8b8b8b;
      }
      th:first-child,
      td:first-child {
        width: 60%;
      }
      button {
        background-color: transparent;
        border: none;
        color: #8b8b8b;
        padding: 0 0.5rem;
        &:focus {
          box-shadow: none;
        }
        svg {
          color: #8b8b8b;
        }
      }
    }
    .detail {
      display: flex;
      justify-content: flex-start;
      .icon {
        max-height: 60px;
        overflow: hidden;
        width: 15%;
        img {
          width: 100%;
        }
      }
      .desc {
        padding: 0 0.5rem;
        width: 85%;
        .title {
          color: #000;
          font-size: 0.875rem;
          margin: 0 0 0.05rem;
        }
        p {
          color: #8b8b8b;
          font-size: 0.75rem;
        }
        .provider {
          color: #000;
          font-size: 0.75rem;
        }
      }
    }
    .footer {
      align-tems: center;
      display: flex;
      justify-content: flex-end;
      padding: 1rem 0;
      width: 100%;
      p {
        color: #404041;
        font-size: 0.875rem;
      }
    }
  }
`;

export default Wrapper;
