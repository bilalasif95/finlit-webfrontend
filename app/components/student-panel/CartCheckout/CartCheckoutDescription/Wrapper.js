import styled from 'styled-components';

const Wrapper = styled.div`
  .cart_checkout {
    margin-top: 31px;
    .header {
      align-tems: center;
      display: flex;
      justify-content: flex-end;
      .input-group {
        width: 30%;
        .form-control {
          border: none;
          border-right: 1px solid #e4e4e4;
          font-size: 1rem;
          padding: 0.25rem 0.75rem;
          &:focus {
            box-shadow: none;
          }
        }
        button {
          background-color: transparent;
          border-color: transparent;
          color: #8b8b8b;
          padding: 0.25rem 0.5rem;
          position: absolute;
          right: 0;
          top: -2px;
          z-index: 4;
          &:focus {
            box-shadow: none;
          }
        }
        .btn-secondary {
          background-color: transparent;
          border-color: transparent;
        }
      }
      .cart {
        align-items: center;
        display: flex;
        justify-content: center;
        width: 12%;
        margin: 6px 0px 30px;
        div {
          &:first-child {
            border-right: 1px solid #e4e4e4;
          }
        }
        .num_cont {
          position: relative;
          width: 2.2rem;
          text-align: center;
          width: 50px;
        }
        svg {
          color: #8b8b8b;
          height: 20px;
          width: 20px;
          cursor: pointer;
        }
        .no_item {
          align-tems: center;
          display: flex;
          justify-content: center;
          background-color: #404041;
          border-radius: 2rem;
          color: #fff;
          font-size: 0.75rem;
          height: 1.25rem;
          padding: 1.1px 0 0;
          position: absolute;
          top: -4px;
          right: 5px;
          width: 1.25rem;
        }
      }
    }
    table {
      width: 100%;
      tr {
        align-tems: center;
        display: flex;
        justify-content: space-between;
        border-bottom: 1px solid #e4e4e4;
      }
      th,
      td {
        font-size: 0.875rem;
        font-weight: 400;
        padding: 0.7rem 0;
        width: 20%;
      }
      th {
        color: #404041;
      }
      td {
        color: #8b8b8b;
      }
      th:first-child,
      td:first-child {
        width: 60%;
      }
      button {
        background-color: transparent;
        border: none;
        color: #8b8b8b;
        padding: 0 0.5rem;
        &:focus {
          box-shadow: none;
        }
        svg {
          color: #8b8b8b;
        }
      }
    }
    .detail {
      display: flex;
      justify-content: flex-start;
      .icon {
        max-height: 60px;
        min-height: 60px;
        min-width: 65px;
        overflow: hidden;
        width: 15%;
        img {
          height: 100%;
          width: 100%;
        }
      }
      .desc {
        padding: 0 0.5rem;
        width: 85%;
        .title {
          color: #000;
          font-size: 0.875rem;
          margin: 0 0 0.05rem;
          overflow: hidden;
          vertical-align: middle;
          text-overflow: ellipsis;
          display: -webkit-box;
          -webkit-line-clamp: 1;
          -webkit-box-orient: vertical;
        }
        p {
          color: #8b8b8b;
          font-size: 0.75rem;
          overflow: hidden;
          vertical-align: middle;
          text-overflow: ellipsis;
          display: -webkit-box;
          -webkit-line-clamp: 1;
          -webkit-box-orient: vertical;
        }
        .provider {
          color: #000;
          font-size: 0.75rem;
        }
      }
    }
    .cartDetailsCont {
      margin-top: 64px;
      .customBorder {
        border-bottom: 1px solid #d8d8d8;
        width: 100%;
        margin: 50px 0px 50px;
        display: block;
      }
      .moreCourses {
        .card {
          border-radius: 10px;
          .card-body {
            padding: 20px 20px 0px 20px;
            .card-title {
              font-size: 20px;
              color: #404041;
              margin-bottom: 20px;
            }
            .cardList {
              display: flex;
              margin-bottom: 35px;
              li {
                color: #404041;
                font-size: 16px;
                font-weight: 700;
                &:not(:last-child) {
                  margin-right: 25px;
                }
                img {
                  height: 30px;
                  width: 30px;
                  margin-right: 10px;
                }
              }
            }
          }
          .card-footer {
            background-color: #fff;
            border-bottom-left-radius: 10px;
            border-bottom-right-radius: 10px;
            display: flex;
            justify-content: space-between;
            align-items: center;
            padding: 10px 20px;
            .price {
              h4 {
                font-size: 20px;
                color: #404041;
                font-weight: 700;
              }
            }
            .addCart {
              display: flex;
              align-items: center;
              button {
                font-size: 14px;
                color: #3e952e;
                font-weight: 700;
                img {
                  margin-right: 6px;
                }
              }
            }
          }
        }
        h2 {
          font-size: 24px;
          color: #404041;
          margin-bottom: 30px;
        }
      }
      .courseDetail {
        padding: 20px 20px;
        border: 1px solid #d8d8d8;
        border-radius: 10px;
        margin-bottom: 20px;
        .checkoutDetails {
          display: flex;
          align-items: center;
          justify-content: space-between;
          width: 100%;
          margin-bottom: 22px;
          .deleteImg {
            position: absolute;
            right: 0;
            top: 0;
            height: 20px;
            width: 16px;
            cursor: pointer;
          }
          .courseDetailImg {
            height: 92px;
            width: 16%;
            img {
              height: 100%;
              object-fit: cover;
              border-radius: 5px;
            }
          }
          .courseDetailText {
            width: 82%;
            position: relative;
            h2 {
              white-space: nowrap;
              overflow: hidden;
              text-overflow: ellipsis;
              width: 230px;
            }
            h2,
            h3 {
              font-size: 20px;
              color: #404041;
              margin-bottom: 0px;
              line-height: 40px;
            }
            p {
              font-size: 14px;
              color: #8b8b8b;
            }
          }
        }
        .courseDetailList {
          display: flex;
          justify-content: space-between;
          align-items: center;
          width: 90%;
          margin-bottom: 0px;
          li {
            color: #404041;
            font-size: 14px;
            font-weight: 700;
            display: flex;
            align-items: center;
            svg {
              margin-right: 10px;
              color: #8b8b8b;
              width: 20px;
              height: 20px;
            }
            img {
              margin-right: 10px;
              width: 20px;
              height: 20px;
            }
          }
        }
      }
    }
    .footer {
      align-tems: center;
      display: flex;
      justify-content: flex-end;
      padding: 1rem 0;
      width: 100%;
      p {
        color: #404041;
        font-size: 0.875rem;
      }
    }
  }
  @media (max-width: 1024px) {
    .cart_checkout {
      .header {
        .cart {
          width: 10%;
        }
      }
    }
  }
  @media (max-width: 991px) {
    .cart_checkout {
      .header {
        .input-group {
          width: 100%;
          .form-control {
            padding: 0.25rem 0.75rem 0.25rem 0;
          }
        }
      }
      table {
        th,
        td {
          min-width: 130px;
          width: 28%;
        }
        th:first-child,
        td:first-child {
          min-width: 270px;
          width: 60%;
        }
        th:last-child,
        td:last-child {
          min-width: 60px;
          width: 12%;
        }
      }
    }
  }
  @media (max-width: 768px) {
    .cart_checkout{
      .cartDetailsCont {
        margin-top: 30px;
      }
      .cartDetailsCont {
        .courseDetail {
          padding: 16px 16px;
          .checkoutDetails {
            .courseDetailImg {
              height: 70px;
              width: 30%;
            }
            .courseDetailText {
              width: 70%;
              h3, h2 {
                font-size: 16px;
                line-height: 30px;
              }
            }
          }
          .courseDetailList {
            width: 100%;
            flex-wrap: wrap;
            li {
              width: 50%;
              margin-bottom: 15px;
            }
          }
        }
        .customBorder {
          margin: 30px 0px 20px;
        }
        .moreCourses {
          h2 {
            font-size: 20px;
            margin-bottom: 20px;
            line-height: 30px;
          }
          .card {
            .card-body { 
              padding: 10px 10px 0px 10px;
              .card-title {
                font-size: 16px;
                margin-bottom: 15px;
              }
              .cardList {
                flex-wrap: wrap;
                margin-bottom: 0px;
                li {
                  font-size: 14px;
                  width: 50%;
                  margin-bottom: 15px;
                  &:first-child {
                    width: 100%;
                  }
                  &:not(last-child) {
                    margin-right: 0px !important;
                  }
                }
              }
            }
            .card-footer {
              padding: 10px 10px;
              .price {
                h4 {
                  font-size: 16px;
                }
              }
              .addCart {
                button {
                  padding: 0px;
                }
              }
            }
          }
        } 
      }
    } 
  }
  @media (max-width: 575px) {
    .cart_checkout {
      table {
        th,
        td {
          width: 28%;
        }
        th:first-child,
        td:first-child {
          width: 60%;
        }
        th:last-child,
        td:last-child {
          width: 12%;
        }
      }
    }
      .cart_checkout {
        margin-top: 0;
        .footer {
          padding: 0 !important;
        }
        .cartDetailsCont {
          .moreCourses {
            .card {
              &:first-child {
                margin-bottom: 16px;
              }
              .card-body {
                .cardList {
                  flex-wrap: nowrap;
                  li {
                    &:not(:last-child) {
                      margin-right: 25px !important;
                    }
                    width: auto !important;
                    &:first-child {
                      width: auto;
                    }
                  }
                }
              }
            }
          }
        }
      }
    }
  @media (max-width: 424px) {
    .cart_checkout {
      .header {
        .cart {
          width: 15%;
        }
      }
    }
  }
  @media (max-width: 375px) {
    .cart_checkout {
      .cartDetailsCont {
        .checkoutDetails {
          display: initial !important;
          .courseDetailImg {
            width: 100% !important;
            height: auto !important;
            margin-bottom: 15px;
            img {
              width: 100%;
            }
          }
          .courseDetailText {
            width: 100% !important;
            margin-bottom: 15px;
            img {
              top: 10px !important;
            }
          }
        }
      }
    }
  }
`;

export default Wrapper;
