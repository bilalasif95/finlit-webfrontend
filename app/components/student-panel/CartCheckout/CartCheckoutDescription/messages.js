/*
 * Cart Checkout Description Messages
 *
 * This contains all the text for the Cart Checkout Description component.
 */
import { defineMessages } from 'react-intl';

export const scope = 'boilerplate.components.CartCheckoutDescription';

export default defineMessages({
  Courses: {
    id: `${scope}.Courses`,
    defaultMessage: `Courses`,
  },
  Price: {
    id: `${scope}.Price`,
    defaultMessage: `Price`,
  },
  Remove: {
    id: `${scope}.Remove`,
    defaultMessage: 'Remove',
  },
  Title: {
    id: `${scope}.Title`,
    defaultMessage: '100 Days of code',
  },
  ShortDesc: {
    id: `${scope}.ShortDesc`,
    defaultMessage: 'Lorem Ipsum',
  },
  ProvidedBy: {
    id: `${scope}.ProvidedBy`,
    defaultMessage: 'Provided By:',
  },
  ProviderName: {
    id: `${scope}.ProviderName`,
    defaultMessage: 'Joel & Natalie Rivera',
  },
  Subtotal: {
    id: `${scope}.Subtotal`,
    defaultMessage: 'Subtotal :',
  },
});
