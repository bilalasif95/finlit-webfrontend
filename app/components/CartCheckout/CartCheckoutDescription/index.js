/*
 * Cart Checkout Description Component
 */
import React from 'react';
import { FormattedMessage } from 'react-intl';
import { BiSearchAlt2 } from 'react-icons/bi';
import { IoMdCart } from 'react-icons/io';
import { AiOutlineDelete } from 'react-icons/ai';
// import { AiFillHeart } from 'react-icons/ai';
// import { Link } from 'react-router-dom';
import { InputGroup, Input, Button } from 'reactstrap';
import messages from './messages';
import Wrapper from './Wrapper';
import reg from '../../../images/reg.png';

function CartCheckoutDescription() {
  const CoursesList = [
    {
      id: 0,
      img: reg,
      alt: 'Courses',
      title: <FormattedMessage {...messages.Title} />,
      desc: <FormattedMessage {...messages.ShortDesc} />,
      providedby: <FormattedMessage {...messages.ProvidedBy} />,
      providername: <FormattedMessage {...messages.ProviderName} />,
      price: <FormattedMessage {...messages.Price} />,
    },
    {
      id: 1,
      img: reg,
      alt: 'Courses',
      title: <FormattedMessage {...messages.Title} />,
      desc: <FormattedMessage {...messages.ShortDesc} />,
      providedby: <FormattedMessage {...messages.ProvidedBy} />,
      providername: <FormattedMessage {...messages.ProviderName} />,
      price: <FormattedMessage {...messages.Price} />,
    },
    {
      id: 2,
      img: reg,
      alt: 'Courses',
      title: <FormattedMessage {...messages.Title} />,
      desc: <FormattedMessage {...messages.ShortDesc} />,
      providedby: <FormattedMessage {...messages.ProvidedBy} />,
      providername: <FormattedMessage {...messages.ProviderName} />,
      price: <FormattedMessage {...messages.Price} />,
    },
    {
      id: 3,
      img: reg,
      alt: 'Courses',
      title: <FormattedMessage {...messages.Title} />,
      desc: <FormattedMessage {...messages.ShortDesc} />,
      providedby: <FormattedMessage {...messages.ProvidedBy} />,
      providername: <FormattedMessage {...messages.ProviderName} />,
      price: <FormattedMessage {...messages.Price} />,
    },
    {
      id: 4,
      img: reg,
      alt: 'Courses',
      title: <FormattedMessage {...messages.Title} />,
      desc: <FormattedMessage {...messages.ShortDesc} />,
      providedby: <FormattedMessage {...messages.ProvidedBy} />,
      providername: <FormattedMessage {...messages.ProviderName} />,
      price: <FormattedMessage {...messages.Price} />,
    },
  ];
  return (
    <Wrapper id="list">
      <div className="cart_checkout">
        <div className="header">
          <InputGroup>
            <Input type="text" name="search" id="seach" placeholder="Search" />
            <Button>
              <BiSearchAlt2 />
            </Button>
          </InputGroup>
          <div className="cart">
            <IoMdCart />
            <span className="no_item">20</span>
          </div>
        </div>
        <div className="table-responsive">
          <table>
            <thead>
              <tr>
                <th>
                  <FormattedMessage {...messages.Courses} />
                </th>
                <th>
                  <FormattedMessage {...messages.Price} />
                </th>
                <th>
                  <FormattedMessage {...messages.Remove} />
                </th>
              </tr>
            </thead>
            <tbody>
              {CoursesList.map(item => (
                <tr key={item.id}>
                  <td>
                    <div className="detail">
                      <div className="icon">
                        <img src={item.img} alt="Icon" />
                      </div>
                      <div className="desc">
                        <div className="title">{item.title}</div>
                        <p>{item.desc}</p>
                        <div className="provider">
                          {item.providedby} {item.providername}
                        </div>
                      </div>
                    </div>
                  </td>
                  <td>$150.00</td>
                  <td>
                    <Button>
                      <AiOutlineDelete />
                    </Button>
                  </td>
                </tr>
              ))}
            </tbody>
          </table>
        </div>
        <div className="footer">
          <div className="sub_total">
            <p>
              <FormattedMessage {...messages.Subtotal} /> $4230
            </p>
          </div>
        </div>
      </div>
    </Wrapper>
  );
}

export default CartCheckoutDescription;
