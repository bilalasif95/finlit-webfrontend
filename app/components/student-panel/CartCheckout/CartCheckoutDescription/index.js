/*
 * Cart Checkout Description Component
 */
import React from 'react';
import { FormattedMessage } from 'react-intl';
import { IoMdCart } from 'react-icons/io';
import { AiOutlineDelete } from 'react-icons/ai';
// import { AiFillHeart } from 'react-icons/ai';
// import { Link } from 'react-router-dom';
import { InputGroup, Input, Button } from 'reactstrap';
import messages from './messages';
import Wrapper from './Wrapper';
import reg from '../../../../images/reg.png';

function CartCheckoutDescription(props) {
  console.log('detailsCart', props.details);

  return (
    <Wrapper id="list">
      <div className="cart_checkout">
        <div className="header">
          {/* <InputGroup>
            <Input type="text" name="search" id="seach" placeholder="Search" />
            <Button>
              <BiSearchAlt2 />
            </Button>
          </InputGroup> */}
          <div className="cart">
            <div className="num_cont">
              <IoMdCart />
              <span className="no_item">
                {props.details &&
                  props.details.items &&
                  props.details.items.length}
              </span>
            </div>
          </div>
        </div>
        <div className="table-responsive">
          <table>
            <thead>
              <tr>
                <th>
                  <FormattedMessage {...messages.Courses} />
                </th>
                <th>
                  <FormattedMessage {...messages.Price} />
                </th>
                <th>
                  <FormattedMessage {...messages.Remove} />
                </th>
              </tr>
            </thead>
            <tbody>
              {props.details &&
                props.details.items &&
                props.details.items.length > 0 &&
                props.details.items.map(item => (
                  <tr key={item.id}>
                    <td>
                      <div className="detail">
                        <div className="icon">
                          <img
                            src={item.product && item.product.eventImage}
                            alt="Icon"
                          />
                        </div>
                        <div className="desc">
                          <div className="title">
                            {item.product && item.product.data.mainTitle}
                          </div>
                          <p>{item.product && item.product.data.subTitle}</p>
                          <div className="provider">
                            {/* {item.providedby} {item.product.data.creator} */}
                          </div>
                        </div>
                      </div>
                    </td>
                    <td>${item.product && item.product.data.price}</td>
                    <td>
                      <Button
                        onClick={() => props.RemoveFromCart(item.cartItemId)}
                      >
                        <AiOutlineDelete />
                      </Button>
                    </td>
                  </tr>
                ))}
            </tbody>
          </table>
        </div>
        <div className="footer">
          <div className="sub_total">
            <p>
              <FormattedMessage {...messages.Subtotal} /> ${props.details.total}
            </p>
          </div>
        </div>
      </div>
    </Wrapper>
  );
}

export default CartCheckoutDescription;
