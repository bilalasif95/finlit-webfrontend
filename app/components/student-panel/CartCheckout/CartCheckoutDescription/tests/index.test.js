import React from 'react';
import renderer from 'react-test-renderer';
import { IntlProvider } from 'react-intl';
import { Provider } from 'react-redux';
import { browserHistory, BrowserRouter } from 'react-router-dom';

import CartCheckoutDescription from '../index';
import configureStore from '../../../../../configureStore';

describe('<CartCheckoutDescription />', () => {
  let store;

  beforeAll(() => {
    store = configureStore({}, browserHistory);
  });

  it('should render and match the snapshot', () => {
    const renderedComponent = renderer
      .create(
        <Provider store={store}>
          <IntlProvider locale="en">
            <BrowserRouter>
              <CartCheckoutDescription details={{ total: 30 }} />
            </BrowserRouter>
          </IntlProvider>
        </Provider>,
      )
      .toJSON();

    expect(renderedComponent).toMatchSnapshot();
  });
});
