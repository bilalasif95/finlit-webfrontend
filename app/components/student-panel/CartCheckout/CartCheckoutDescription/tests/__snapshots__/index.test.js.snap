// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CartCheckoutDescription /> should render and match the snapshot 1`] = `
<div
  className="Wrapper-yoca5k-0 kBiCOS"
  id="list"
>
  <div
    className="cart_checkout"
  >
    <div
      className="header"
    >
      <div
        className="input-group"
      >
        <input
          className="form-control"
          id="seach"
          name="search"
          placeholder="Search"
          type="text"
        />
        <button
          aria-label={null}
          className="btn btn-secondary"
          onClick={[Function]}
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 24 24"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M19.023,16.977c-0.513-0.488-1.004-0.997-1.367-1.384c-0.372-0.378-0.596-0.653-0.596-0.653l-2.8-1.337 C15.34,12.37,16,10.763,16,9c0-3.859-3.14-7-7-7S2,5.141,2,9s3.14,7,7,7c1.763,0,3.37-0.66,4.603-1.739l1.337,2.8 c0,0,0.275,0.224,0.653,0.596c0.387,0.363,0.896,0.854,1.384,1.367c0.494,0.506,0.988,1.012,1.358,1.392 c0.362,0.388,0.604,0.646,0.604,0.646l2.121-2.121c0,0-0.258-0.242-0.646-0.604C20.035,17.965,19.529,17.471,19.023,16.977z M9,14 c-2.757,0-5-2.243-5-5s2.243-5,5-5s5,2.243,5,5S11.757,14,9,14z"
            />
          </svg>
        </button>
      </div>
      <div
        className="cart"
      >
        <div
          className="num_cont"
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 512 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M169.6 377.6c-22.882 0-41.6 18.718-41.6 41.601 0 22.882 18.718 41.6 41.6 41.6s41.601-18.718 41.601-41.6c-.001-22.884-18.72-41.601-41.601-41.601zM48 51.2v41.6h41.6l74.883 151.682-31.308 50.954c-3.118 5.2-5.2 12.482-5.2 19.765 0 27.85 19.025 41.6 44.825 41.6H416v-40H177.893c-3.118 0-5.2-2.082-5.2-5.2 0-1.036 2.207-5.2 2.207-5.2l20.782-32.8h154.954c15.601 0 29.128-8.317 36.4-21.836l74.882-128.8c1.237-2.461 2.082-6.246 2.082-10.399 0-11.446-9.364-19.765-20.8-19.765H135.364L115.6 51.2H48zm326.399 326.4c-22.882 0-41.6 18.718-41.6 41.601 0 22.882 18.718 41.6 41.6 41.6S416 442.082 416 419.2c0-22.883-18.719-41.6-41.601-41.6z"
            />
          </svg>
          <span
            className="no_item"
          >
            20
          </span>
        </div>
      </div>
    </div>
    <div
      className="table-responsive"
    >
      <table>
        <thead>
          <tr>
            <th>
              <span>
                Courses
              </span>
            </th>
            <th>
              <span>
                Price
              </span>
            </th>
            <th>
              <span>
                Remove
              </span>
            </th>
          </tr>
        </thead>
        <tbody>
          <tr>
            <td>
              <div
                className="detail"
              >
                <div
                  className="icon"
                >
                  <img
                    alt="Icon"
                    src="IMAGE_MOCK"
                  />
                </div>
                <div
                  className="desc"
                >
                  <div
                    className="title"
                  >
                    <span>
                      100 Days of code
                    </span>
                  </div>
                  <p>
                    <span>
                      Lorem Ipsum
                    </span>
                  </p>
                  <div
                    className="provider"
                  >
                    <span>
                      Provided By:
                    </span>
                     
                    <span>
                      Joel & Natalie Rivera
                    </span>
                  </div>
                </div>
              </div>
            </td>
            <td>
              $150.00
            </td>
            <td>
              <button
                aria-label={null}
                className="btn btn-secondary"
                onClick={[Function]}
              >
                <svg
                  fill="currentColor"
                  height="1em"
                  stroke="currentColor"
                  strokeWidth="0"
                  style={
                    Object {
                      "color": undefined,
                    }
                  }
                  viewBox="0 0 1024 1024"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M360 184h-8c4.4 0 8-3.6 8-8v8h304v-8c0 4.4 3.6 8 8 8h-8v72h72v-80c0-35.3-28.7-64-64-64H352c-35.3 0-64 28.7-64 64v80h72v-72zm504 72H160c-17.7 0-32 14.3-32 32v32c0 4.4 3.6 8 8 8h60.4l24.7 523c1.6 34.1 29.8 61 63.9 61h454c34.2 0 62.3-26.8 63.9-61l24.7-523H888c4.4 0 8-3.6 8-8v-32c0-17.7-14.3-32-32-32zM731.3 840H292.7l-24.2-512h487l-24.2 512z"
                  />
                </svg>
              </button>
            </td>
          </tr>
          <tr>
            <td>
              <div
                className="detail"
              >
                <div
                  className="icon"
                >
                  <img
                    alt="Icon"
                    src="IMAGE_MOCK"
                  />
                </div>
                <div
                  className="desc"
                >
                  <div
                    className="title"
                  >
                    <span>
                      100 Days of code
                    </span>
                  </div>
                  <p>
                    <span>
                      Lorem Ipsum
                    </span>
                  </p>
                  <div
                    className="provider"
                  >
                    <span>
                      Provided By:
                    </span>
                     
                    <span>
                      Joel & Natalie Rivera
                    </span>
                  </div>
                </div>
              </div>
            </td>
            <td>
              $150.00
            </td>
            <td>
              <button
                aria-label={null}
                className="btn btn-secondary"
                onClick={[Function]}
              >
                <svg
                  fill="currentColor"
                  height="1em"
                  stroke="currentColor"
                  strokeWidth="0"
                  style={
                    Object {
                      "color": undefined,
                    }
                  }
                  viewBox="0 0 1024 1024"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M360 184h-8c4.4 0 8-3.6 8-8v8h304v-8c0 4.4 3.6 8 8 8h-8v72h72v-80c0-35.3-28.7-64-64-64H352c-35.3 0-64 28.7-64 64v80h72v-72zm504 72H160c-17.7 0-32 14.3-32 32v32c0 4.4 3.6 8 8 8h60.4l24.7 523c1.6 34.1 29.8 61 63.9 61h454c34.2 0 62.3-26.8 63.9-61l24.7-523H888c4.4 0 8-3.6 8-8v-32c0-17.7-14.3-32-32-32zM731.3 840H292.7l-24.2-512h487l-24.2 512z"
                  />
                </svg>
              </button>
            </td>
          </tr>
          <tr>
            <td>
              <div
                className="detail"
              >
                <div
                  className="icon"
                >
                  <img
                    alt="Icon"
                    src="IMAGE_MOCK"
                  />
                </div>
                <div
                  className="desc"
                >
                  <div
                    className="title"
                  >
                    <span>
                      100 Days of code
                    </span>
                  </div>
                  <p>
                    <span>
                      Lorem Ipsum
                    </span>
                  </p>
                  <div
                    className="provider"
                  >
                    <span>
                      Provided By:
                    </span>
                     
                    <span>
                      Joel & Natalie Rivera
                    </span>
                  </div>
                </div>
              </div>
            </td>
            <td>
              $150.00
            </td>
            <td>
              <button
                aria-label={null}
                className="btn btn-secondary"
                onClick={[Function]}
              >
                <svg
                  fill="currentColor"
                  height="1em"
                  stroke="currentColor"
                  strokeWidth="0"
                  style={
                    Object {
                      "color": undefined,
                    }
                  }
                  viewBox="0 0 1024 1024"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M360 184h-8c4.4 0 8-3.6 8-8v8h304v-8c0 4.4 3.6 8 8 8h-8v72h72v-80c0-35.3-28.7-64-64-64H352c-35.3 0-64 28.7-64 64v80h72v-72zm504 72H160c-17.7 0-32 14.3-32 32v32c0 4.4 3.6 8 8 8h60.4l24.7 523c1.6 34.1 29.8 61 63.9 61h454c34.2 0 62.3-26.8 63.9-61l24.7-523H888c4.4 0 8-3.6 8-8v-32c0-17.7-14.3-32-32-32zM731.3 840H292.7l-24.2-512h487l-24.2 512z"
                  />
                </svg>
              </button>
            </td>
          </tr>
          <tr>
            <td>
              <div
                className="detail"
              >
                <div
                  className="icon"
                >
                  <img
                    alt="Icon"
                    src="IMAGE_MOCK"
                  />
                </div>
                <div
                  className="desc"
                >
                  <div
                    className="title"
                  >
                    <span>
                      100 Days of code
                    </span>
                  </div>
                  <p>
                    <span>
                      Lorem Ipsum
                    </span>
                  </p>
                  <div
                    className="provider"
                  >
                    <span>
                      Provided By:
                    </span>
                     
                    <span>
                      Joel & Natalie Rivera
                    </span>
                  </div>
                </div>
              </div>
            </td>
            <td>
              $150.00
            </td>
            <td>
              <button
                aria-label={null}
                className="btn btn-secondary"
                onClick={[Function]}
              >
                <svg
                  fill="currentColor"
                  height="1em"
                  stroke="currentColor"
                  strokeWidth="0"
                  style={
                    Object {
                      "color": undefined,
                    }
                  }
                  viewBox="0 0 1024 1024"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M360 184h-8c4.4 0 8-3.6 8-8v8h304v-8c0 4.4 3.6 8 8 8h-8v72h72v-80c0-35.3-28.7-64-64-64H352c-35.3 0-64 28.7-64 64v80h72v-72zm504 72H160c-17.7 0-32 14.3-32 32v32c0 4.4 3.6 8 8 8h60.4l24.7 523c1.6 34.1 29.8 61 63.9 61h454c34.2 0 62.3-26.8 63.9-61l24.7-523H888c4.4 0 8-3.6 8-8v-32c0-17.7-14.3-32-32-32zM731.3 840H292.7l-24.2-512h487l-24.2 512z"
                  />
                </svg>
              </button>
            </td>
          </tr>
          <tr>
            <td>
              <div
                className="detail"
              >
                <div
                  className="icon"
                >
                  <img
                    alt="Icon"
                    src="IMAGE_MOCK"
                  />
                </div>
                <div
                  className="desc"
                >
                  <div
                    className="title"
                  >
                    <span>
                      100 Days of code
                    </span>
                  </div>
                  <p>
                    <span>
                      Lorem Ipsum
                    </span>
                  </p>
                  <div
                    className="provider"
                  >
                    <span>
                      Provided By:
                    </span>
                     
                    <span>
                      Joel & Natalie Rivera
                    </span>
                  </div>
                </div>
              </div>
            </td>
            <td>
              $150.00
            </td>
            <td>
              <button
                aria-label={null}
                className="btn btn-secondary"
                onClick={[Function]}
              >
                <svg
                  fill="currentColor"
                  height="1em"
                  stroke="currentColor"
                  strokeWidth="0"
                  style={
                    Object {
                      "color": undefined,
                    }
                  }
                  viewBox="0 0 1024 1024"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M360 184h-8c4.4 0 8-3.6 8-8v8h304v-8c0 4.4 3.6 8 8 8h-8v72h72v-80c0-35.3-28.7-64-64-64H352c-35.3 0-64 28.7-64 64v80h72v-72zm504 72H160c-17.7 0-32 14.3-32 32v32c0 4.4 3.6 8 8 8h60.4l24.7 523c1.6 34.1 29.8 61 63.9 61h454c34.2 0 62.3-26.8 63.9-61l24.7-523H888c4.4 0 8-3.6 8-8v-32c0-17.7-14.3-32-32-32zM731.3 840H292.7l-24.2-512h487l-24.2 512z"
                  />
                </svg>
              </button>
            </td>
          </tr>
        </tbody>
      </table>
    </div>
    <div
      className="footer"
    >
      <div
        className="sub_total"
      >
        <p>
          <span>
            Subtotal :
          </span>
           $4230
        </p>
      </div>
    </div>
  </div>
</div>
`;
